test/unit/BLSRegistryCoordinatorWithIndicesUnit.t.sol

```diff
@@ -579,6 +579,23 @@ contract BLSRegistryCoordinatorWithIndicesUnit is MockAVSDeployer {
         );
     }
 
+    // @notice verify that it is possible for an operator to register, deregister, and then register again!
+    function testReregisterOperatorWithCoordinator_Valid() public {
+        testDeregisterOperatorWithCoordinatorForSingleQuorumAndSingleOperator_Valid();
+
+        uint32 registrationBlockNumber = 201;
+
+        bytes memory quorumNumbers = new bytes(1);
+        quorumNumbers[0] = bytes1(defaultQuorumNumber);
+
+        cheats.startPrank(defaultOperator);
+        
+        cheats.roll(registrationBlockNumber);
+        
+        registryCoordinator.registerOperatorWithCoordinator(quorumNumbers, defaultPubKey, defaultSocket);
+
+        uint256 quorumBitmap = BitmapUtils.orderedBytesArrayToBitmap(quorumNumbers);
```

Comment:
did you want to make an assertion about this

Reply:
ahh yeah good call, one sec I'll add one

Reply:
@gpsanant check this commit https://github.com/Layr-Labs/eigenlayer-middleware/pull/20/commits/aa1782de7e7b167706e3d1d7adfc82f3e7c5bbc9